admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -124,9 +124,15 @@ to the redis group::
 
   usermod -a -G redis www-data
 
-You might need to restart apache for the changes to take effect::
+And modify the ``unixsocketperm`` of the ``redis.conf`` accordingly::
+
+  unixsocketperm 770
+
+You might need to restart apache and redis for the changes to take effect::
 
  systemctl restart apache2
+ systemctl restart redis-server
+ systemctl restart redis
```

Comment:
I just installed `redis-server` on ubuntu. In this case, you have `redis-server` as service. `redis` in this case is just an alias and this way you would restart redis twice:
![image](https://user-images.githubusercontent.com/8111200/166214070-df1f567c-5926-4f3f-9cc4-9142684f10ed.png)

Let's just keep the redis-server.